sdk/service/backend.go

```diff
@@ -73,6 +74,19 @@ func (s *Backend) querier(request cosmostypes.Request, path []string, req abci.R
 		return s.Get(request, hash)
 	case "list":
 		return s.List(request)
+	case "hash":
```

Comment:
does it need to be a backend function? the sdk could do it no?

Reply:
Don't we want to have everthing on the backend at some point?

Reply:
it could but then we have some logic on two different places which is hard to manage

Reply:
@NicolasMahe suggested to have the hash calculation on the sdk and not backend file to avoid the call to the abci, let's think about that, I like to have all the business logic on the backend at least for now until we really see ways to improve it but at least it centralizes the "processing" part.
Anyway let's merge and think about that later